tests/mpi/periodic_neighbor_01.mpirun=3.output

```diff
@@ -0,0 +1,343 @@
+DEAL::All of the cells with periodic neighbors on rank 2
```

Comment:
I don't understand why you appear to only get output from processor 2. I guess that you might want to use 'MPILogInitAll' here, see e.g. in `tests/mpi/mg_01.cc` for how to use it.